packages/edit-site/src/components/sidebar/global-styles-sidebar.js

```diff
@@ -197,7 +197,7 @@ export default function GlobalStylesSidebar( {
 		>
 			<TabPanel
 				tabs={ [
-					{ name: 'global', title: __( 'Global' ) },
+					{ name: 'root', title: __( 'Root' ) },
```

Comment:
Similarly, exposing the term "Root" to non technical end users in the sidebar may be confusing.

Reply:
+100

Reply:
The idea we discussed in https://github.com/WordPress/gutenberg/issues/28163 is that we have a lot of things that use the name "global", so it losses its meaning and has created confusion when talking about this. The intent is to walk away from to global to a different name as to bring some clarity in conversations:

- global sidebar vs block sidebar
- the typography panel of the root block in the global sidebar
- the color panel of the paragraph block in the block sidebar
- etc

I'm not particularly attached to the `root` name. Happy to hear options. `site` could work, although it has different connotations to me (your whole site opposed to a particular "block").